Process.c

```diff
@@ -273,8 +273,13 @@ void Process_printTime(RichString* str, unsigned long long totalHundredths, bool
 
 void Process_fillStarttimeBuffer(Process* this) {
    struct tm date;
+   time_t now = time(NULL);
    (void) localtime_r(&this->starttime_ctime, &date);
-   strftime(this->starttime_show, sizeof(this->starttime_show) - 1, (this->starttime_ctime > (time(NULL) - 86400)) ? "%R " : "%b%d ", &date);
+
+   strftime(this->starttime_show,
+            sizeof(this->starttime_show) - 1,
+            (this->starttime_ctime > now - 86400) ? "%R " : (this->starttime_ctime > now - 364*86400) ? "%b%d " : " %Y ",
```

Comment:
Wait, isn't `%b%d` for the second case actually 2 parameters, but we are providing only one?

Reply:
@BenBE Don't confuse this format with `printf` format. For `strftime` it takes only one `struct tm *` parameter as the source data to print.

Reply:
Sorry, my bad. My preference of the more verbose variant with the 3 explicit branches stand nonetheless. ;-)